src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        <p className="Person__age">
+          {age ? `I am ${age}` : null}
+        </p>
```

Comment:
```suggestion
        {age && (
          <p className="Person__age">
            `I am ${age}`
          </p>
        )
```
Зараз ти рендерший пустий тег p, якщо немає age. Давай взагалі його рендерити тільки тоді, коли age є